@xen-orchestra/web/src/pages/host/[id]/network.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <div class="host-network-view">
```

Comment:
Be sure to match the component's name:
```suggestion
  <div class="network">
```